packages/terra-clinical-application/README.md

```diff
@@ -1,8 +1,8 @@
 # Terra Clinical Application
 
 
-[![NPM version](http://img.shields.io/npm/v/terra-clinical-application.svg)](https://www.npmjs.org/package/terra-clinical-application)
-[![Build Status](https://travis-ci.org/cerner/terra-clinical.svg?branch=master)](https://travis-ci.org/cerner/terra-clinical)
+[![NPM version](http://img.shields.io/npm/v/terra-clinical-application)](https://www.npmjs.org/package/terra-clinical-application)
```

Comment:
It looks like the NPM version is failing and falling back, should these link be pulling from https://badgen.net/npm/v/?

https://github.com/cerner/terra-core/pull/1793/files#diff-2a1313e8f0a12dc4041616e117137137R4

![image](https://user-images.githubusercontent.com/6720991/44746946-7d303c00-aad1-11e8-9923-7288d2191a88.png)